nx.json

```diff
@@ -184,7 +184,7 @@
     },
     "code-pushup-eslint": {
       "cache": true,
-      "inputs": ["code-pushup-inputs", "lint-eslint-inputs"],
```

Comment:
We do not generate the artifacts in this task and have the inputs in the dependent lint task